plugins/multiforward.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: multiforward
+spec:
+  version: v0.0.1
+  homepage: https://github.com/njnygaard/kubectl-multiforward
+  shortDescription: Port Forward to multiple Kubernetes Services
+  description: |
+    This plugin allows you to forward to multiple Kubernetes Services
+    without the use of multiple terminal windows.
+  caveats: |
+    * You should have ~/.multiforward.yaml or ./.multiforward.yaml configured
```

Comment:
Like so?
```suggestion
    You should have ~/.multiforward.yaml or ./.multiforward.yaml configured. See `multiforward --help` for format details.
```

Reply:
I feel dumb. Sorry for not understanding. I updated to this (before I realized I could have committed your suggested change right from here).